nvtabular/inference/triton/ensemble.py

```diff
@@ -178,10 +179,7 @@ def export_hugectr_ensemble(
     hugectr_params,
     name,
     output_path,
-    label_columns,
```

Comment:
Similar to the TF call - you will need to update the hugectr movielens/criteo inference notebooks here too, not just to remove the passed in label_columns but also the cats/conts

Reply:
Yep - looks like the tests catch these issues